Sources_v3/StreamChatUI/Utils/CGFloat+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+extension CGFloat {
+    static func scaledFontSize(for size: CGFloat) -> CGFloat {
+        UIFontMetrics.default.scaledValue(for: size)
+    }
+}
```

Comment:
AFAIR that doesn't really work so lets get rid of it

Reply:
This shouldn't be needed